src/components/Navbar.js

```diff
@@ -4,30 +4,33 @@ import { Link } from 'react-router-dom';
 import logo from '../images/Logo.png';
 import '../styles/Navbar.css';
 
-const Navbar = () => {
+const Navbar = (props) => {
+
+  const { pages } = props;
+
   return (
     <div>
       <nav className="Navbar">
-        <div className="Navbar-Logo-Image">
-          <img src={logo} alt="Logo" />;
+
+        {/* Wrap link around logo to return you to the landing page */}
+        <Link to="/" className="Navbar-Logo-Link">
+          <div className="Navbar-Logo-Image">
+            <img src={logo} alt="Logo" />;
         </div>
-        <div className="Navbar-Logo">Auzzie Aid</div>
+          <div className="Navbar-Logo">Auzzie Aid</div>
+        </Link>
         <ul className="Navbar-Menu">
-          <li className="Navbar-Menu-ListItem">
-            <Link to="/" className="Navbar-Menu-ListItem-Link">
-              Live Updates
-            </Link>
-          </li>
-          <li className="Navbar-Menu-ListItem">
-            <Link to="/info" className="Navbar-Menu-ListItem-Link">
-              Info
-            </Link>
-          </li>
-          <li className="Navbar-Menu-ListItem">
-            <Link to="/donate" className="Navbar-Menu-ListItem-Link">
-              Donate
-            </Link>
-          </li>
+
+          {/* Dynamically render links based on the pages array */}
+          {pages.map((page) => {
+            if (page.type.name !== pages[0].type.name && page.type.name !== pages[pages.length - 1].type.name) {
+              return (
+                <li key={page.type.name} className="Navbar-Menu-ListItem" onClick={() => { console.log(`You Clicked ${page.type.name}`) }}>
+                  <Link to={`/${page.type.name}`} className="Navbar-Menu-ListItem-Link">{page.type.name}</Link>
+                </li >
```

Comment:
I think I have an idea for this where we can extend the pages object in App.js and leverage this new functionality! Great job my man 💪